core/src/main/java/org/apache/accumulo/core/metadata/schema/Ample.java

```diff
@@ -398,9 +399,7 @@ ConditionalTabletMutator requireOperation(TabletOperation operation,
   /**
    * Convenience interface for handling conditional mutations with a status of UNKNOWN.
    */
-  interface UknownValidator {
-    boolean shouldAccept(TabletMetadata tabletMetadata);
-  }
+  interface UknownValidator extends Predicate<TabletMetadata> {}
```

Comment:
Should this typo be fixed?

```suggestion
  interface UnknownValidator extends Predicate<TabletMetadata> {}
```

Reply:
updated in c5fe3e2